server/serverAPI/controller/InventoryController.ts

```diff
@@ -34,9 +87,12 @@ export default class InventoryController extends BaseUserController {
     getAll = async (req: Request, res: Response) => {
         let parameters = new Map<string, any>([["username", req.serverUser.username]]);
 
-        return this.requestGet(parameters, res).then(user => {
+        try {
+            let user = await this.requestGet(parameters, res)
```

Comment:
Do you need to check that the lookup for user succeeds? Same comment everywhere you look up a user.

Reply:
requestGet logic will throw if it doesn't succeed which will be handled by try catch block